mlos_bench/mlos_bench/launcher.py

```diff
@@ -208,11 +221,18 @@ def _load_config(self,
             global_config["config_path"] = config_path
         return global_config
 
-    def _load_tunable_values(self, args_tunables: Optional[str]) -> TunableGroups:
+    def _init_tunable_values(self, random_init: bool, seed: Optional[int],
+                             args_tunables: Optional[str]) -> TunableGroups:
         """
-        Load key/value pairs of the tunable values from given JSON files, if any.
+        Initialize the tunables and load key/value pairs of the tunable values
+        from given JSON files, if specified.
         """
         tunables = self.environment.tunable_params
+
+        if random_init:
+            opt = MockOptimizer(tunables=tunables, service=None, config={"seed": seed})
```

Comment:
Creating an optimizer and throwing it away just to get random values?
Do you need to pass "random_init" to the MockOptimizer config?

Reply:
that's the easiest way to do things IMO. Technically, we can factor out that random initialization and use it in both the MockOptimizer and here, but maybe I'll do it in the next PR.

Reply:
also, no, we don't need to pass `random_init` to `MockOptimizer` - it is always random (except for the first iteration when `start_with_defaults` is True)